src/store/wizardSlice.ts

```diff
@@ -127,6 +128,7 @@ const initialState: wizardState = {
   fileSystem: {
     mode: 'automatic',
     partitions: [],
+    hasErrorOnSubmit: false,
```

Comment:
I'm not really comfortable with the idea of keeping error state in the store, it feels like that is something we should be figuring out using a validator or helper function where needed.

So in this case we need to know both in the custom footer (to decide whether or not to disable the next button) and in the component (to decide whether to render the error messages). I think that should be possible without adding a new field to the store's wizardSlice.

Let's remove this and see if we can get it working without it.

Reply:
We agree to change the name to isNextButtonPristine and initialize to true, and when the user presses on the 'Next' button this variable will change to false, and will not return to true.
if there are duplicate rows after the user presses on the next button at the first time, the error message will show immediately.